app/views/projects/edit.html.erb

```diff
@@ -16,7 +17,7 @@
                 Edit volunteer position <%= @project.name %>
                 </h3>
                 <p class="mt-1 max-w-2xl text-sm leading-5 text-gray-500">
-                This information will be displayed publicly.
+                This information will be displayed publicly. If you have any questions about creating a position on this platform, please contact newhavenpandemic@gmail.com.
```

Comment:
We should probably use the `HWC_EMAIL` config variable or some other config instead of hard coding the emails.